data.sql

```diff
@@ -52,4 +52,55 @@ WHERE
 	OR
 	(own.fullname = 'Melody Pond' AND (anm.name = 'Charmander' OR anm.name = 'Squirtle' OR anm.name = 'Blossom'))
 	OR
-	(own.fullname = 'Dean Winchester' AND (anm.name = 'Angemon' OR anm.name = 'Boarmon'));
\ No newline at end of file
+	(own.fullname = 'Dean Winchester' AND (anm.name = 'Angemon' OR anm.name = 'Boarmon'));
+
+
+INSERT INTO
+	vets(name, age, date_of_graduation)
+	VALUES
+		('William Tatcher', 45, '2000-04-23'),
+		('Maisy Smith', 26, '2019-01-17'),
+		('Stephanie Mendez', 64, '1981-05-04'),
+		('Jack Harkness', 38, '2008-06-08');
+
+
+INSERT INTO
+	species(name, age, date_of_graduation)
+	VALUES
+		('William Tatcher', 45, '2000-04-23'),
+		('Maisy Smith', 26, '2019-01-17'),
+		('Stephanie Mendez', 64, '1981-05-04'),
+		('Jack Harkness', 38, '2008-06-08');
+
+
+INSERT INTO
+	specializations(species_id, vet_id)
+	VALUES
+		(1, 1),
+		(1, 3),
+		(2, 3),
+		(2,4);
+
```

Comment:
* [OPTIONAL] This insertion could have been done dynamically where you do not know the `vet_id` and `specie_id`.

```sql
 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'William Tatcher'), (SELECT id FROM species WHERE name = 'Pokemon'));

 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Pokemon'));

 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Digimon'));

 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Jack Harkness'), (SELECT id FROM species WHERE name = 'Digimon'));
```